update-server-repo-setup/generate-repos-config

```diff
@@ -180,7 +180,11 @@ class RepoConfigGenerator:
                                 }
                             )
                     else:
-                        additional_repos = product_settings['additional_repos']
+                        additional_repos = product_settings.get(
```

Comment:
nit-picking

```python3
additional_repos = product_settings.get(
     'additional_repos'
)
if not additional_repos:
    continue
```
could become
```python3
for additional_repo in product_settings.get('additional_repos', []):
```

in any case, we could benefit of the default value of the dict when the key is not found

```python3
additional_repos = product_settings.get(
     'additional_repos', []
)
for additional_repo in additional_repos:
```
probably you preferred explicitness for the reader, which is good, as I said nit-picking

Reply:
Agree on the default of empty string which then removes the need for the nonesy check

Reply:
Thanks, fixed.